pkgs/by-name/az/azote/package.nix

```diff
@@ -0,0 +1,124 @@
+{
+  lib,
+  fetchFromGitHub,
+  gobject-introspection,
+  gtk3,
+  libappindicator-gtk3,
+  nix-update-script,
+  python3,
+  wrapGAppsHook3,
+
+  # runtime
+  bash,
+  findutils,
+  gawk,
+  imagemagick,
+  procps,
+
+  # wayland
+  waylandSupport ? true,
+  sway,
+  swaybg,
+  wlr-randr,
+  grim,
+  slurp,
+
+  # x11
+  x11Support ? true,
+  feh,
+  maim,
+  slop,
+  xorg,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "azote";
+  version = "1.12.9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "nwg-piotr";
+    repo = "azote";
+    rev = "v${version}";
+    hash = "sha256-ffymC1pxR6v6Qn0n9W3vENh4IGHZk1WNQog+ohuNI68=";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    python3.pkgs.setuptools
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    gtk3
+    libappindicator-gtk3
+  ];
+
+  prePatch =
+    # FIXME: not replacing full lines due to escaping issues
```

Comment:
What's the escaping issue? Preferably, we should fix this before merging.

Reply:
> Should I use a patch file instead?

No, `swaybg -o '{}' -i \"{}\" -m {} &` can be matchable but you only need to replace that part. You might be able to use sed for that instead.

Reply:
But you shouldn't need to be patching it like this unless it uses a full path. You're wrapping it and you change `$PATH` so this shouldn't even be required.

Reply:
> azote creates a script file at ~/.azotebg, which isn't wrapped by Nix

Does azote launch the script itself?